configuring-infrastructure-components/deadlines.md

```diff
@@ -33,18 +39,39 @@ Cancelling a deadline could for example come in handy when a certain event means
  but the client payed the amount which means that the deadline is obsolete and can be canceled).
 
 ```java
-deadlineManager.cancelSchedule("myDeadline", deadlineId);
+class DeadlineCancelingComponent {
+    void cancelMyDeadline(String deadlineId) {
+        deadlineManager.cancelSchedule("myDeadline", deadlineId);
+    }
+}
 ```
 
-> **Canceling Deadlines**
->
-> It is possible to cancel all deadlines of a given name by invoking `deadlineManager.cancelAll("myDeadline")`.
+There are more option to cancel a deadline nexted to through the received `deadlineId` and the used `deadlineName`:
 
-If you need some contextual data about the Deadline during the Deadline Handling,
- you can attach a Deadline Payload when scheduling a Deadline:
+ * `cancelAll(String)`
+  Cancels _every_ scheduled deadline matching the given deadline name.
+  Note that this thus also cancels deadlines from other aggregate and/or saga instances matching the name.
+ * `cancelAllWithinScope(String)`
```

Comment:
```suggestion
 * `cancelAllWithinScope(String name)`
```

Reply:
I'll add the parameter name, good point. I'll do `deadlineName` though, to be a tad more specific.